Setup/customer-example.env

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
You don't actually need a shebang in a script being sourced by another script.

Reply:
Do you mean we don't need the first line with #!/bin/bash ?

Reply:
Yes. It is ignored in files that are sourced.